presto-hive-function-namespace/src/main/java/com/facebook/presto/hive/functions/FunctionRegistry.java

```diff
@@ -0,0 +1,495 @@
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFRegExp;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFRestrictInformationSchema;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFRound;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFRpad;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFSQCountCheck;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFSentences;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFSha2;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFSize;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFSortArray;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFSortArrayByField;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFSoundex;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFSplit;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFStringToMap;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFStruct;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFSubstringIndex;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFTimestamp;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToBinary;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToChar;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToDate;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToDecimal;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToIntervalDayTime;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToIntervalYearMonth;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToTimestampLocalTZ;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToUnixTimeStamp;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToUtcTimestamp;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFToVarchar;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFTranslate;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFTrim;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFTrunc;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFUnion;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFUnixTimeStamp;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFUpper;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFWhen;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDFWidthBucket;
+import org.apache.hadoop.hive.ql.udf.generic.UDFCurrentDB;
+import org.apache.hadoop.hive.ql.udf.xml.GenericUDFXPath;
+import org.apache.hadoop.hive.ql.udf.xml.UDFXPathBoolean;
+import org.apache.hadoop.hive.ql.udf.xml.UDFXPathDouble;
+import org.apache.hadoop.hive.ql.udf.xml.UDFXPathFloat;
+import org.apache.hadoop.hive.ql.udf.xml.UDFXPathInteger;
+import org.apache.hadoop.hive.ql.udf.xml.UDFXPathLong;
+import org.apache.hadoop.hive.ql.udf.xml.UDFXPathShort;
+import org.apache.hadoop.hive.ql.udf.xml.UDFXPathString;
+import org.apache.hadoop.hive.serde.serdeConstants;
+
+public final class FunctionRegistry
+{
+    public static final String LEAD_FUNC_NAME = "lead";
```

Comment:
nits: these static variables are probably not necessary. Either use static for all function names, or just inline all of them.